src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -633,7 +633,7 @@ private AssetData GetManifestAsAsset(IImmutableList<AssetData> assets, string lo
                 {
                     Location = location,
                 }),
-                Name = $"assets/manifests/{azDORepoName}/{AssetVersion}/{manifestFileName}",
+                Name = $"assets/manifests/{repoName}/{AssetVersion}/{manifestFileName}",
```

Comment:
I think this will end up with <Blob Id="assets/manifests/NuGet/NuGet.Client//MergedManifest.xml" NonShipping="true" /> for nuget.client's github repo.

Reply:
Yep, just verified with a repo that builds directly from GitHub.

Build.Repository.Name will be `<org>/<repo>` so we can't use the variable directly.